packages/react-ui-core/src/Counter/Counter.js

```diff
@@ -109,6 +109,8 @@ export default class Counter extends PureComponent {
             className
           )}
         {...props}
+        // NOTE: ensure click is noop so handler is only on counter spans
+        onClick={() => {}}
```

Comment:
If this is what we really want to do, I think it would be more appropriate to add it to the destructure above instead.  Doing this here creates an unnecessary `onClick` event on every counter.